include/SDL3/SDL_video.h

```diff
@@ -130,6 +130,7 @@ typedef enum
     SDL_WINDOW_KEYBOARD_GRABBED     = 0x00100000,   /**< window has grabbed keyboard input */
     SDL_WINDOW_VULKAN               = 0x10000000,   /**< window usable for Vulkan surface */
     SDL_WINDOW_METAL                = 0x20000000,   /**< window usable for Metal view */
+    SDL_WINDOW_TRANSPARENT          = 0x40000000,   /**< window with transparent buffer */
```

Comment:
I don't want to bikeshed names too much but I feel like `SDL_WINDOW_ALLOW_TRANSPARENCY` might be more representative of what it does, since it needs content rendered to the window's surface to work. Or there are a bunch of other possibilities (`SDL_WINDOW_FRAMEBUFFER_TRANSPARENT` etc.)

Reply:
Yes, I'll change that once we find a good naming

Reply:
I actually like this name, but we can discuss as we get more refined and understand how other platforms work.